### PR TITLE
Week 4 project

### PR DESCRIPTION
**Pull request by wlalanne**

### CODE REVIEW

File: week_4/project/week_4.py [hunk 1]
```diff
@@ -5,22 +5,59 @@
...
-    # Use your op logic from week 3 (you will need to make a slight change)
-    pass
-
-
-get_s3_data_docker, process_data_docker, put_redis_data_docker = with_resources()
+@asset(
+    description="List of Stocks",
+    config_schema={"s3_key": str},
+    required_resource_keys={"s3"},
+    compute_kind="s3",
```
Comment (pskip-unlearn) [inline week_4/project/week_4.py:12]: ooh, what does `compute_kind` do?

**Review by pskip-unlearn — APPROVED** [review]
LGTM!

**Comment by pskip-unlearn** [comment]
Great job

File: week_4/project/week_4.py (hunk 1 above)
Comment (wlalanne) [reply week_4/project/week_4.py:12]: It adds a badge in Dagit that represents the kind of computation that produces the asset. I think it's mostly for organization.

**Pull request merged**